src/Sylius/Bundle/ApiBundle/DataPersister/AddressBookDataPersister.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataPersister;
+
+use ApiPlatform\Core\DataPersister\ContextAwareDataPersisterInterface;
+use Sylius\Bundle\ApiBundle\Provider\CustomerProvider;
+use Sylius\Component\Core\Model\AddressInterface;
+use Sylius\Component\User\Model\UserInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+final class AddressBookDataPersister implements ContextAwareDataPersisterInterface
+{
+    /** @var ContextAwareDataPersisterInterface */
+    private $decoratedDataPersister;
+
+    /** @var TokenStorageInterface */
+    private $tokenStorage;
+
+    /** @var CustomerProvider */
+    private $customerProvider;
+
+    public function __construct(
+        ContextAwareDataPersisterInterface $decoratedDataPersister,
+        TokenStorageInterface $tokenStorage,
+        CustomerProvider $customerProvider
+    ) {
+        $this->decoratedDataPersister = $decoratedDataPersister;
+        $this->tokenStorage = $tokenStorage;
+        $this->customerProvider = $customerProvider;
+    }
+
+    public function supports($data, array $context = []): bool
+    {
+        return $data instanceof AddressInterface;
+    }
+
+    public function persist($data, array $context = [])
```

Comment:
```suggestion
    public function persist($data, array $context = [])
```
Is possible to add return type?

Reply:
i dont really know xD, interface doesnt have any return type to it